packages/linux-firmware/linux-firmware.spec

```
@@ -53,10 +49,471 @@ mkdir -p %{buildroot}/%{fwdir}/updates
+%{fwdir}/i915/skl_dmc_ver1_27.bin.zst
+%{fwdir}/i915/skl_guc_32.0.3.bin.zst
+%{fwdir}/i915/skl_guc_33.0.0.bin.zst
+%{fwdir}/i915/skl_guc_49.0.1.bin.zst
+%{fwdir}/i915/skl_guc_62.0.0.bin.zst
+%{fwdir}/i915/skl_guc_69.0.3.bin.zst
+%{fwdir}/i915/skl_guc_70.1.1.bin.zst
+%{fwdir}/i915/skl_guc_ver1.bin.zst
+%{fwdir}/i915/skl_guc_ver4.bin.zst
+%{fwdir}/i915/skl_guc_ver6.bin.zst
+%{fwdir}/i915/skl_guc_ver6_1.bin.zst
+%{fwdir}/i915/skl_guc_ver9_33.bin.zst
+%{fwdir}/i915/skl_huc_2.0.0.bin.zst
+%{fwdir}/i915/skl_huc_ver01_07_1398.bin.zst
+%{fwdir}/i915/tgl_dmc_ver2_04.bin.zst
+%{fwdir}/i915/tgl_dmc_ver2_06.bin.zst
+%{fwdir}/i915/tgl_dmc_ver2_08.bin.zst
+%{fwdir}/i915/tgl_dmc_ver2_12.bin.zst
+%{fwdir}/i915/tgl_guc_35.2.0.bin.zst
+%{fwdir}/i915/tgl_guc_49.0.1.bin.zst
+%{fwdir}/i915/tgl_guc_62.0.0.bin.zst
+%{fwdir}/i915/tgl_guc_69.0.3.bin.zst
+%{fwdir}/i915/tgl_guc_70.1.1.bin.zst
+%{fwdir}/i915/tgl_guc_70.bin.zst
+%{fwdir}/i915/tgl_huc.bin.zst
+%{fwdir}/i915/tgl_huc_7.0.12.bin.zst
+%{fwdir}/i915/tgl_huc_7.0.3.bin.zst
+%{fwdir}/i915/tgl_huc_7.5.0.bin.zst
+%{fwdir}/i915/tgl_huc_7.9.3.bin.zst
+%{fwdir}/i915/xe2lpd_dmc.bin.zst
+%{fwdir}/i915/xe3lpd_*.bin.zst
+
+# Intel network firmware
+%{fwdir}/intel/ice/ddp-comms/ice_comms-1.3.55.0.pkg.zst
+%{fwdir}/intel/ice/ddp-wireless_edge/ice_wireless_edge-1.3.23.0.pkg.zst
+%{fwdir}/intel/ice/ddp/ice-1.3.43.0.pkg.zst
+%{fwdir}/intel/ice/ddp/ice.pkg.zst
+
+# Network firmware - QLogic FastLinQ (qed)
+%{fwdir}/qed/qed_init_values-8.10.9.0.bin.zst
+%{fwdir}/qed/qed_init_values-8.14.6.0.bin.zst
+%{fwdir}/qed/qed_init_values-8.18.9.0.bin.zst
+%{fwdir}/qed/qed_init_values-8.20.0.0.bin.zst
+%{fwdir}/qed/qed_init_values-8.30.12.0.bin.zst
+%{fwdir}/qed/qed_init_values-8.33.12.0.bin.zst
+%{fwdir}/qed/qed_init_values-8.37.7.0.bin.zst
+%{fwdir}/qed/qed_init_values-8.40.33.0.bin.zst
+%{fwdir}/qed/qed_init_values_zipped-8.10.10.0.bin.zst
+%{fwdir}/qed/qed_init_values_zipped-8.10.5.0.bin.zst
+%{fwdir}/qed/qed_init_values_zipped-8.15.3.0.bin.zst
+%{fwdir}/qed/qed_init_values_zipped-8.20.0.0.bin.zst
+%{fwdir}/qed/qed_init_values_zipped-8.33.1.0.bin.zst
+%{fwdir}/qed/qed_init_values_zipped-8.33.11.0.bin.zst
+%{fwdir}/qed/qed_init_values_zipped-8.37.2.0.bin.zst
+%{fwdir}/qed/qed_init_values_zipped-8.37.7.0.bin.zst
+%{fwdir}/qed/qed_init_values_zipped-8.4.2.0.bin.zst
+%{fwdir}/qed/qed_init_values_zipped-8.42.2.0.bin.zst
+%{fwdir}/qed/qed_init_values_zipped-8.59.1.0.bin.zst
+%{fwdir}/qed/qed_init_values_zipped-8.7.3.0.bin.zst
+
+# Network firmware - Broadcom Tigon3 (tg3)
+%{fwdir}/tigon/tg3.bin.zst
+%{fwdir}/tigon/tg357766.bin.zst
+%{fwdir}/tigon/tg3_tso.bin.zst
+%{fwdir}/tigon/tg3_tso5.bin.zst
+
+# License files
+%license GPL-2
+%license GPL-3
+%license LICENCE.chelsio_firmware
+%license LICENCE.myri10ge_firmware
+%license LICENCE.phanfw
+%license LICENSE.i915
+%license LICENSE.ice
+%license LICENSE.ice_enhanced
+%license WHENCE
 %{_cross_attribution_file}
+
+%files amdgpu
+%license LICENSE.amdgpu
+%dir %{fwdir}
```

This subpackage should not own the same directory as its parent:

```suggestion
%dir %{fwdir}/amdgpu
%{fwdir}/amdgpu/*
```